single_tech_samples/storage/sample1_data_lifecycle/terraform/main.tf

```diff
@@ -0,0 +1,9 @@
+module "datalifecycle" {
```

Comment:
As this is just a wrapper, I suggest removing the whole module folder and add all the files here directly.

Reply:
Done